packages/embark/package.json

```diff
@@ -34,8 +34,7 @@
   ],
   "scripts": {
     "build": "cross-env BABEL_ENV=node babel src --copy-files --extensions \".js,.ts\" --out-dir dist --root-mode upward --source-maps",
-    "ci": "npm-run-all qa ci:*",
-    "ci:coverage-report": "nyc report --reporter=text-lcov | coveralls || exit 0",
+    "ci": "npm run qa",
```

Comment:
If this is just an alias, should we get rid of one of them?

Reply:
Having `ci` separate from `qa` allows us to do things in Azure Pipelines (previously Travis and Appveyor) differently than we do them in the of context preparing a release.

The names don't have a deep meaning in relation to each other, I just needed different script names for similar but distinct purposes.

The only place that matters right now is `embark-ui` — in Azure Pipelines we don't do the production CRA build as part of its `ci` script (because it takes a long time), but when preparing a release we do run that build.

In the future we maybe can work out a better system.